### ISSUE TITLE
Ports must be open?

### ISSUE THREAD

**Issue by Freddyqt** [synfinatic/netflow2ng#28]
Hello
docker run synfinatic/netflow2ng
Run the command and it automatically showed me the following output:
level=info msg="Starting netflow2ng"
level=info msg="Started ZMQ listener on: tcp://*:5556"
level=info msg="Listening on UDP 0.0.0.0:2055" Type=NetFlow

Then run the ntop command, to start ntop on port 5556
Collecting flows on tcp://172.16.88.7:5556

It showed me the following line, but in the web browser, it doesn't show any result.. my sopgos xg firewall directs netflow traffic to port 2055 of my server.. did i miss something? I opened the udp ports and I can't get it to work.

Some help?

Thank you

**Comment by Freddyqt** [comment]
Port 8080
I started it via dockerfile and got this error on the port.
404 page not found

The command was
docker run -p 5556:5556 -p 2055:2055 -p 8080:8080 net

**Comment by synfinatic** [comment]
Look at https://github.com/synfinatic/netflow2ng/blob/main/docker-compose.yaml

TL;DR: you need to specify that 2055 is UDP, because default for docker is TCP.

**Comment by Freddyqt** [comment]
Hello friend, I run the port in udp with Dockerfile and it throws me the following error.

level=error msg="Error from: NetFlow (0) duration: 6.408µs. Unknown NetFlow version 5 (only decodes v9 and v10/IPFIX
level=error msg="Error from: NetFlow (0) duration: 6.408µs. Unknown NetFlow version 5 (only decodes v9 and v10/IPFIX
level=error msg="Error from: NetFlow (0) duration: 6.408µs. Unknown NetFlow version 5 (only decodes v9 and v10/IPFIX
level=error msg="Error from: NetFlow (0) duration: 6.408µs. Unknown NetFlow version 5 (only decodes v9 and v10/IPFIX

do you know what it is for?

**Comment by synfinatic** [comment]
NetFlow v5 is not supported.
